tree_for_tractometry.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+usage() { echo "$(basename $0) [-r RBX] [-t tractoflow/results] [-o output]" 1>&2; exit 1; }
```

Comment:
I think we could have noddi and fw here too as an optionnal argument. In the for loop we could check if the argument -n or -f is empty or not. What do you think @mdesco ?

Reply:
sure but my bash skills are too weak to attempt this... go for it if you can.

Reply:
Ok I will do it based on you scripts.